### PR TITLE
Blurry-Effect-Interface Project

### PR DESCRIPTION
**Pull request by gaurav24072002**
## Related Issue
  - Separated the .html file in public folder , .css and .js in assest folder.
  - allow this project edit by maintainers.  

Closes: #225 , #169 

### Describe the changes you've made
  - Separated the .html file in public folder , .css and .js in assest folder.
  - allow this project edit by maintainers.  

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA
### Screenshots

**BEFORE -**
Screenshot (107)

**AFTER -**
Screenshot (108)

### CODE REVIEW

**Comment by gaurav24072002** [comment]
@urvashi-code1255  and @Vishal-raj-1  now is this good to go.. I have solved the issues.

**Review by urvashi-code1255 — COMMENTED** [review]
@gaurav24072002 Add your project details in script.js

Also pull the recent changes.

**Comment by gaurav24072002** [comment]
ok sure ...

**Comment by gaurav24072002** [comment]
Please review it and tell me if some more changes are required or not. 
Thank you @urvashi-code1255 for reviewing my  commits as well.

File: assets/js/blurry_loading_interface.js [hunk 1]
```diff
@@ -13,4 +13,10 @@ function blurring(){
...
     bg.style.filter = `blur(${30-(load/100)*30}px)`
-}
\ No newline at end of file
+}
+// This project is based on the starter page of any website which is a Blurry effect with a countdown from 0% to 100%.
+// The structure is made from blurry_loading_interface.html file.
+// To butify The project we use the blurry_loading_interface.css file.
+// The main brain of the body that is how will the opacity change from 100% to 0% is written under this javascript file that is blurry_loading_interface.js file.
+// You can use this repository to add a starter page to your website and showcase it to your friends or for company basis as well.
+// Thank you guys for viewing this project...
```
Comment (gaurav24072002) [inline assets/js/blurry_loading_interface.js:22]: Please tell me if I need to add more details to it or not like telling how the code base works and all that stuffs...

**Comment by gaurav24072002** [comment]
@urvashi-code1255 and @Vishal-raj-1  can you please merge my pull request and add tag of gssoc '21 on it so that it could be counted in gssoc '21 ..and Please tell me if further changes are to be made in it or not.. thank you..

**Comment by gaurav24072002** [comment]
and please assign me this project in assignee section 😅

**Comment by gaurav24072002** [comment]
@urvashi-code1255  Any Suggestions that I need to do to improve it ??

**Comment by gaurav24072002** [comment]
Any Updates for this issue ??

**Review by urvashi-code1255 — COMMENTED** [review]
@gaurav24072002 You don't have to put your project details like this.

Open script.js once and see that every project is described there in some points like gif, name etc.

**Comment by gaurav24072002** [comment]
Ya sure I got it ...

**Comment by gaurav24072002** [comment]
Now is It Good to go ?

**Review by urvashi-code1255 — COMMENTED** [review]
@gaurav24072002 Add GIF of your project!!

Also remove the horizontal and vertical scroll bar from your project

**Comment by gaurav24072002** [comment]
ok

**Comment by gaurav24072002** [comment]
Now is it good to go?

**Comment by urvashi-code1255** [comment]
@gaurav24072002 Please resolve the merge conflicts again

**Comment by gaurav24072002** [comment]
@urvashi-code1255  any Update ?

**Comment by urvashi-code1255** [comment]
@gaurav24072002 Resolve it again.

**Comment by gaurav24072002** [comment]
Hey! @urvashi-code1255 I have been working with other projects also but they didn't took so much of time to merge an issue.
you are just again and again making me resolve conflicts. Please merge this time. and please add tag of Gssoc '21 to it and level tag.

**Review by ridsuteri — APPROVED** [review]
good work @gaurav24072002 ,thanks for contributing :)

**Comment by gaurav24072002** [comment]
Thank you so much @ridsuteri  👍

**Comment by ridsuteri** [comment]
@gaurav24072002 :)

**Pull request merged**